src/vmm/src/device_manager/persist.rs

```diff
@@ -17,7 +17,6 @@ use super::mmio::*;
 use crate::arch::DeviceType;
```

Comment:
I don't understand the point of the commit `refactor(block): remove indirection in imports`.

It looks like it does two things:
1. remove `VhostUserBlock` variant from error type `DevicePersistError`.
2. it re-exports types from `virtio::device`, `virtio::metrics` and `virtio::persist` as types of the `virtio::block` module.
3. It makes modules `io` and `event_handler` public

I do not understand how 2 and 3 are useful. Actually, I think we should avoid doing 2, because we should really be able to differentiate between virtio and vhost-user cases. Regarding 3, I don't see why we should do that.

Finally, I think 1. should not be part of this commit.

Reply:
1. indeed can be done in previous commits
2. This is done to create an initial platform where `Block` will exist. As I just looked, it seems `virtio::metrics` is not used in the end, so I will remove it.
3. Initially as I remember they needed to be made public due to the changes. Seems `io` does not need to be in the end. `metrics` needs to be thought for logger

Reply:
done